mosquitto/templates/deployment.yaml

```
@@ -16,6 +16,7 @@ spec:
         {{ toYaml . | indent 8 | trim }}
     {{- end }}
       serviceAccountName: {{ include "mosquitto.serviceAccountName" . }}
+      securityContext: {{- toYaml .Values.podSecurityContext | nindent 8 }}
```

Only render if `podSecurityContext` is set.

```suggestion
    {{- with .Values.podSecurityContext }}
      securityContext:
        {{ toYaml . | indent 8 | trim }}
    {{- end }}
```